src/main/java/com/fasterxml/jackson/dataformat/xml/deser/XmlTokenStream.java

```diff
@@ -371,21 +371,23 @@ private final int _next() throws XMLStreamException
     
     private final String _collectUntilTag() throws XMLStreamException
     {
-        String text = null;
+        if (_xmlReader.isEmptyElement()) {
+            _xmlReader.next();
+            return null;
+        }
+
+        StringBuilder text = new StringBuilder();
```

Comment:
Oh. Actually, I see. So, this was to get empty String over `null`.
I think I'll modify this bit differently, after first merging,.